### DISCUSSION TITLE
Let users decide which badges to display

### DISCUSSION THREAD

**Discussion by tomoveu** [community/community#65987] [Other Feature Feedback, Questions, & Ideas] [answered]
### Select Topic Area

Product Feedback

### Body

GitHub badges are fun and at the same time mandatory all-on or all-off. I think users should be enabled to choose which badges to display on their GitHub profiles and which not :)

What do you think?

**Comment by radar-blip** [comment, marked as answer]
We have this option.
1. Click on the achievements link just above your badges.
2. Click on whichever badge you want to hide (or show).
3. Click on the eye icon in the upper left corner.

**Note:** you may have to refresh the page for it to take effect.

![hide](https://github.com/community/community/assets/39196767/d30d262d-0d47-4be7-9c2c-e60a8f279d29)

**Reply by chzfabian** [reply]
brital wick

**Comment by HamzaGorcevic** [comment]
We can decide wich to show or to hide ...
